sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -56,7 +56,8 @@ public class ChatClient {
             throw AzureError.client("Unable to form base URL.")
         }
 
-        let authPolicy = CommunicationUserCredentialPolicy(credential: credential)
+        let communicationCredential = CommunicationPolicyTokenCredential(credential)
```

Comment:
Just to clarify - if this change is checked into the feature branch, and then the feature branch is updated with master, all that should change is the rename of CommunicationUserCredential -> CommunicationTokenCredential? 

Otherwise if it helps to update the feature branch first we can do that

Reply:
Yes the only change we will need to make is the rename. There shouldn't be any other conflicts